src/scss/components/paragraphs/_chart.scss

```diff
@@ -0,0 +1,25 @@
+$_chart-aspect-ratio: 1.78; // 16/9
+
+.chart {
+  figure {
```

Comment:
I would combine these two selectors to a single line selector to keep the nesting at minimum.